doc-files/release-notes-19.md

```diff
@@ -0,0 +1,91 @@
+# Release Notes for JavaFX 19
+
+## Introduction
+
+The following notes describe important changes and information about this release. In some cases, the descriptions provide links to additional detailed information about an issue or a change.
+
+As of JDK 11 the JavaFX modules are delivered separately from the JDK. These release notes cover the standalone JavaFX 19 release. JavaFX 19 requires JDK 11 or later.
+
+## List of Enhancements
+
+Issue key|Summary|Subcomponent
+---------|-------|------------
+[JDK-8274771](https://bugs.openjdk.java.net/browse/JDK-8274771)|Map, FlatMap and OrElse fluent bindings for ObservableValue|base
+[JDK-8283346](https://bugs.openjdk.java.net/browse/JDK-8283346)|Optimize observable ArrayList creation in FXCollections|base
+[JDK-8286552](https://bugs.openjdk.java.net/browse/JDK-8286552)|TextFormatter: UpdateValue/UpdateText is called, when no ValueConverter is set|controls
+[JDK-8268225](https://bugs.openjdk.java.net/browse/JDK-8268225)|Support :focus-visible and :focus-within CSS pseudoclasses|graphics
+[JDK-8277309](https://bugs.openjdk.java.net/browse/JDK-8277309)|Add support for H.265/HEVC to HTTP Live Streaming|media
+
+## List of Fixed Bugs
+
+Issue key|Summary|Subcomponent
+---------|-------|------------
+[JDK-8290331](https://bugs.openjdk.java.net/browse/JDK-8290331)|Binding value left null when immediately revalidated in invalidation listener|base
+[JDK-8290393](https://bugs.openjdk.java.net/browse/JDK-8290393)|Code sample in javadoc of ObservableValue flatMap is incorrect|base
```

Comment:
Same for this one, introduced in 19 and fixed in 19.

Reply:
Hmm. This one especially seems not useful to mention in the release notes. @johanvos can decide whether or not to include it.

Reply:
This is different to me indeed, as it only touches documentation. I'll remove this one.
In the future, it might be good to have a label for these kinds of issues that (1) fix issues caused in the same release version and (2) do not have a runtime impact; so that the algorithm for creating release notes does not require human judgement. Unless we want to explicitly discuss some of the items -- which I'm not strongly opposed to either.

Reply:
I agree with this reasoning. The best way to note this is to use the `Introduced In Version` field. The filter could exclude bugs with a `noreg-doc` label and an `Introduced In Version` that is the same as the `fixVersion`.

Reply:
@johanvos In order to automate this, I propose adding the following logic to the release notes JBS filter:

```
AND ("Introduced In Version" is EMPTY OR "Introduced In Version" != openjfx19 OR
     labels is EMPTY OR labels not in (noreg-doc))
```

Does this look OK to you?

Reply:
yes, that looks like a good algorithm.

Reply:
I've made the change (and added `Introduced In Version = openjfx19` to 8290393).